docs/members-example/views/MemberEditView.tsx

```diff
@@ -80,7 +80,8 @@ export class MemberEditView extends React.Component<IMemberEditViewProps, IMembe
           pin: true
         }]}
         isOpen={this.props.isOpen}
-        onToggle={(isOpen: boolean) => this.props.toggleMemberOpen(isOpen)}>
+        onToggle={(isOpen: boolean) => this.props.toggleMemberOpen(isOpen)}
+        isModal={true}>
```

Comment:
You pass boolean props just as you would on html elements `<Popover {...otherProps} isModal>`